packages/types/src/lib/types.ts

```diff
@@ -226,6 +226,21 @@ export interface LitContractContext {
   StakingBalances: LitContract;
 }
 
+// Only includes the defined contract names, omitting the index signature
+type DefinedContractNamesOnly = Pick<LitContractContext, 'Allowlist'> &
+  Pick<LitContractContext, 'LITToken'> &
+  Pick<LitContractContext, 'Multisender'> &
+  Pick<LitContractContext, 'PKPHelper'> &
+  Pick<LitContractContext, 'PKPNFT'> &
+  Pick<LitContractContext, 'PKPNFTMetadata'> &
+  Pick<LitContractContext, 'PKPPermissions'> &
+  Pick<LitContractContext, 'PubkeyRouter'> &
+  Pick<LitContractContext, 'RateLimitNFT'> &
+  Pick<LitContractContext, 'Staking'> &
+  Pick<LitContractContext, 'StakingBalances'>;
```

Comment:
Why not the other way around?
```suggestion
export interface ExclusiveLitContractContext {
  Allowlist: LitContract;
  LITToken: LitContract;
  Multisender: LitContract;
  PKPHelper: LitContract;
  PKPNFT: LitContract;
  PKPNFTMetadata: LitContract;
  PKPPermissions: LitContract;
  PubkeyRouter: LitContract;
  RateLimitNFT: LitContract;
  Staking: LitContract;
  StakingBalances: LitContract;
}

export interface LitContractContext extends ExclusiveLitContractContext {
  [index: string]: string | any;
}
```

Reply:
Yea, I tried that, but the IntelliSense doesn't pick it up. I also tried to omit string from the interface but will give me the following

<img width="420" alt="image" src="https://github.com/user-attachments/assets/42e70709-9848-42d2-bf31-be8e0dee6408">